templater/templater.go

```diff
@@ -185,6 +186,15 @@ func templateFuncs(c *context.Context, rs *context.ResourceSet) template.FuncMap
 		b, _ := json.Marshal(data)
 		return string(b)
 	}
+
+	m["toYaml"] = func(v interface{}) string {
```

Comment:
I'm still somewhat concerned with the "footgun potential" of having a YAML-serialising function - most Kontemplate outputs are not intended for human consumptions, and for machines it is much more useful to have inline JSON.

I'll need to discuss this with a few people before making a decision.